eng/pipelines/templates/jobs/vmr-synchronization.yml

```diff
@@ -52,16 +56,20 @@ jobs:
       targetRef: ${{ parameters.targetRef }}
 
   # For official builds, push the changes to the VMR
-  # Push only main and release branches
+  # Push main and release branches to the public VMR
   - ${{ if and(not(parameters.noPush), not(in(variables['Build.Reason'], 'PullRequest')), eq(variables['System.TeamProject'], 'internal')) }}:
     - script: |
         set -x
-        git config --global user.email 'dotnet-maestro[bot]@users.noreply.github.com' && git config --global user.name 'dotnet-maestro[bot]'
-        git remote add dotnet 'https://$(BotAccount-dotnet-bot-repo-PAT)@github.com/dotnet/dotnet.git'
-        git fetch dotnet
-        git branch ${{ parameters.vmrBranch }}
-        git branch --set-upstream-to=dotnet/${{ parameters.vmrBranch }} ${{ parameters.vmrBranch }} || echo 'Branch ${{ parameters.vmrBranch }} not found in remote'
-        git push dotnet ${{ parameters.vmrBranch }}
-      displayName: Push changes to dotnet/dotnet
-      workingDirectory: $(vmrPath)
+        ./.dotnet/dotnet darc vmr push --vmr "$(vmrPath)" --commit-verification-pat '$(BotAccount-dotnet-maestro-bot-no-scopes-PAT)' --branch '${{ parameters.vmrBranch }}' --remote-url '$(vmrPublicUrl)' --github-pat '$(BotAccount-dotnet-bot-repo-PAT)' --verbose
```

Comment:
Please
1. Remove `set -x`
2. Change it to `- script: >`
3. Put each parameter on a new line, like:
    ```
    ./.dotnet/dotnet darc vmr push
        --vmr "$(vmrPath)"
        --commit-verification-pat '$(BotAccount-dotnet-maestro-bot-no-scopes-PAT)' 
        --branch '${{ parameters.vmrBranch }}'
        ...
    ```

It will be easier to read